aiotractive/api.py

```diff
@@ -75,13 +76,29 @@ async def request(self, *args, **kwargs):
         except Exception as error:
             raise TractiveError from error
 
+    async def aps_request(self, *args, **kwargs):
+        """Perform request to APS API with error wrapping."""
+        try:
+            return await self.raw_request(*args, base_url=self.APS_API_URL, **kwargs)
+        except ClientResponseError as error:
+            if error.status in [401, 403]:
+                raise UnauthorizedError from error
+            if error.status == 404:
+                raise NotFoundError from error
+            raise TractiveError from error
+        except Exception as error:
+            raise TractiveError from error
+
     async def raw_request(  # pylint: disable=too-many-arguments
-        self, uri, params=None, data=None, method="GET", attempt: int = 1
+        self, uri, params=None, data=None, method="GET", attempt: int = 1, base_url=None
```

Comment:
I would use `API_URL` as the default value for `base_url`.
```suggestion
        self, uri, params=None, data=None, method="GET", attempt: int = 1, base_url=API_URL
```
This allows you to get rid of this block of code:

```python
        if base_url is None:
            base_url = self.API_URL
```